admin/src/main/resources/io/buoyant/admin/js/src/router_clients.js

```diff
@@ -13,12 +13,13 @@ define([
     var EXPAND_CLIENT_THRESHOLD = 6;
 
     function assignColorsToClients(colors, clients) {
-      var colorIdx = 0;
+      var clientMapping = {};
 
-      return _.reduce(clients, function(clientMapping, client) {
-        clientMapping[client.label] = colors[colorIdx++ % colors.length];
-        return clientMapping;
-      }, {});
+      _(clients).sortBy('label').forEach(function(client, idx) {
```

Comment:
there's a `return _.sortBy(clients, 'label');` on line 204 of `routers.js`. is this sorting twice?